app/controllers/v1/supported_ssgs_controller.rb

```diff
@@ -17,7 +17,7 @@ def metadata(opts = {})
       {
         meta: {
           total: opts[:total],
-          revision: resource.revision
+          datastream_revision: resource.datastream_revision
```

Comment:
Wouldn't this break the API?

Reply:
Oh right, it would.. I could just leave the datastream revision method named `revision`? But still add the new `ansible_tasks_revision` method? That'd be easier and fewer updates.

Reply:
The left side should stay as it is, for compatibility.